src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -935,6 +935,24 @@ Default to alert color for visibility titles used within an alert context.
     content: attr(data-placeholder);
 }
 
+/* features page */
+
+.features-page-img {
+    height: 75%;
+    width: 75%;
```

Comment:
Use `max-width: 75%` instead of `width: 75%` (as discussed [above](https://github.com/TEAMMATES/teammates/pull/7539#discussion_r127916762)). Remove `height` as height is set to `auto` from `img-responsive`. To show that this is meant to be used together with `img-responsive`, change the selector to `.img-responsive.features-page-img`.